Changelog v1.8.x.md

```diff
@@ -11,6 +12,10 @@
 - Fix: Incorrect version check with VersionChecker mod integration
 - Fix: Cage Trap owner collision is incorrect
 - Fix: Inventory Scanners can connect to Inventory Scanners by different owners
+- Fix: Bouncing Betty doesn't play a sound when triggered and doesn't jump as high as expected
+- Fix: Block Mines let items and the owner go through
+- Fix: Trophy system destroys projectile even if it has already been removed
+- Fix: Textures of UBM and UOC are transparent at the sides
```

Comment:
> What did you do to the textures to fix the models?
I noticed (and I noticed it only on the UBM and the UOC, if you think more items have this issue, please tell me them) that you could see through the sides of those tools. This was because there were pixels that were almost invisible due to their high alpha value, but they were there and thus Minecraft rendered the sides of the tools like the most outside pixels of the tool, and as these pixes were in some cases invisible, the outline was, too. All I did was clearing out the invisible pixels from the item texture.